.gitignore

```diff
@@ -2,6 +2,6 @@
 .idea
 build
 *.iml
-gradle.properties
 local.properties
 .DS_Store
+*.swp
```

Comment:
IMO, vim swp files should be part of a users global gitignore, rather than the repos, but if this helps y'all's workflow, go for it.

Reply:
Yeah that's a good point. This does add extra protection for the project as a whole. Really just put it in there off of muscle memory.

Reply:
Removed.